docs/module-types.md

```diff
@@ -5,20 +5,20 @@ Both system.js and s.js support loading javascript modules, json modules, and cs
 ## Limitations
 
 The browser spec calls for checking a module's mime type to know whether it is JSON, CSS, or Javascript. SystemJS does not do that,
-since it has to choose upfront whether to append a script element (for js modules) or make a fetch request (for a JSON/CSS module).
+since it has to choose upfront whether to append a script element (for JS modules) or make a fetch request (for a JSON/CSS module).
 So instead of the mime type, the file extension is used to determine the type of the module.
 
-## Javascript modules
-
-SystemJS supports loading javascript modules that are in the following formats:
+SystemJS supports loading modules that are in the following formats:
 
 | Module Format | s.js | system.js |
 | ------------- | ---- | --------- |
 | [System.register](/docs/system-register.md) | :heavy_check_mark: | :heavy_check_mark: |
 | Global variable | [global extra](/README.md#extras) | :heavy_check_mark: |
-| [ESM](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Modules) | [transform extra](/README.md#extras) | [transform extra](/README.md#extras) |
 | [AMD](https://github.com/amdjs/amdjs-api/wiki/AMD) | [AMD extra](/README.md#extras) | [AMD extra](/README.md#extras) |
 | [UMD](https://github.com/umdjs/umd) | [AMD extra](/README.md#extras) | [AMD extra](/README.md#extras) |
+| [Web Assembly](https://github.com/WebAssembly/esm-integration/tree/master/proposals/esm-integration) | :heavy_check_mark: | :heavy_check_mark: |
```

Comment:
Very nitpicky, but it might look better visually if it were organized so that it transitioned from (1) both s.js and system.js support it, to (2) Only system.js has it and s.js needs an extra, to (3) both s.js and system.js need an extra.

In other words, just move all the ones with double check marks up to the top of the list.

<img width="708" alt="image" src="https://user-images.githubusercontent.com/5524384/63654837-e4141f80-c73c-11e9-8d44-c2fd2b0c56b6.png">